tests/system/security/package_vuln_scan.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import subprocess
+import json
+import re
+import os
+from datetime import datetime
+
+INPUT_FILE_PATH_TMP = 'input_file_tmp.txt'
+
+
+def get_args():
+    """
+    Summary:
+        Command line argument parsing method.
+    """
+    parser = argparse.ArgumentParser()
+    input_group = parser.add_mutually_exclusive_group(required=True)
+    input_group.add_argument('-r', dest='input', type=str,
+                             help='specify requirements file path.')
+    input_group.add_argument('-p', dest='pip_mode', action='store_true',
+                             help='enable pip scan mode.')
+    parser.add_argument('-o', dest='output', type=str, required=True,
+                        help='specify output file.')
```

Comment:
By default, the script should provide the information through the standard output and if specified, it should be sent to a file